### PR TITLE
fixes meal pantries button toggle bug

### LINKED ISSUE #2078: Bug: pantry and meal filter button on advance filter feature currently doesn't work
pantry and meal filter button on advance filter feature currently doesn't work

### PR DESCRIPTION
**Pull request by qiqicodes**
fixes #2078 

fixed desktop view

https://github.com/hackforla/food-oasis/assets/68657634/6a6414fc-6f14-4aec-9935-cf99223590a9

fixed mobile view

https://github.com/hackforla/food-oasis/assets/68657634/8019fdbc-866d-4f89-8b46-41c7ec8630f3

### CODE REVIEW

File: client/src/components/FoodSeeker/SearchResults/ResultsMap/ResultsMap.js [hunk 1]
```diff
@@ -204,21 +204,21 @@ const ResultsMap = (
           </Source>
         )}
       </Map>
-      {!loading && hasAdvancedFilterFeatureFlag && isMobile && (
+      {!loading && hasAdvancedFilterFeatureFlag && (
         <Grid
-          container
-          wrap="nowrap"
+          container={isMobile}
+          className={!isMobile ? "advanced-filters-class" : ""}
```
Comment (hanapotski) [inline client/src/components/FoodSeeker/SearchResults/ResultsMap/ResultsMap.js:210]: what are we using this class for?

Comment (qiqicodes) [reply client/src/components/FoodSeeker/SearchResults/ResultsMap/ResultsMap.js:210]: Nice catch. Thanks Hannah, I think we can get rid of this, this class was with the deleted Grid component.

**Comment by hanapotski** [comment]
buttons are working 🙌  but maybe try rebasing 🤔 I pulled this branch and it might be missing the changes from your other PR 👀 

https://github.com/hackforla/food-oasis/assets/25972384/c494e72c-849e-4bc6-a4bf-b135bf80bf19

**Comment by qiqicodes** [comment]
Latest Desktop view:

https://github.com/hackforla/food-oasis/assets/68657634/8c65669b-44c6-4810-bf45-4cec3d2713ed


Latest Toggle function on Desktop view:

https://github.com/hackforla/food-oasis/assets/68657634/03015c63-096f-4123-804d-e9575f6486f2

**Review by hanapotski — APPROVED** [review]
🙌

**Pull request merged**